vendor/github.com/openshift/api/config/v1/types_feature.go

```diff
@@ -45,7 +45,7 @@ var (
 	// your cluster may fail in an unrecoverable way.
 	CustomNoUpgrade FeatureSet = "CustomNoUpgrade"
 
-	// TopologyManager enables ToplogyManager support. Upgrades are enabled with this feature.
+	// TopologyManager enables TopologyManager support. Upgrades are enabled with this feature.
```

Comment:
This code is vendored, so it needs to be changed in openshift/api and then re-vendored.